assets/css/admin.rtl.css

```diff
@@ -17,10 +17,6 @@
 #adminmenu
 	#toplevel_page_wc-admin-path--payments-connect
 	.menu-icon-generic
-	div.wp-menu-image::before,
```

Comment:
probably roll it back, not sure
I also have these changes after local build but I never committed them

Reply:
Neah. They are safe and probably an oversight to commit some time back. If you [look](https://github.com/Automattic/woocommerce-payments/blob/ab3b8baee11b4e2f2e5115536c65889bb6500cd4/assets/css/admin.css#L13-L21) at `admin.css`, the CSS selectors are the same.